app/Model/Reservation.php

```diff
@@ -8,4 +8,81 @@
 class Reservation extends Model
 {
     use SoftDeletes;
+    
+    /**
+     * Declare table
+     *
+     * @var string $tabel table name
+     */
+    protected $table = 'reservations';
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array $fillable
+     */
+    protected $fillable = [
+        'status',
+        'room_id',
+        'target',
+        'target_id',
+        'request',
+        'quantity',
+        'checkin_date',
+        'checkout_date'
+    ];
+
+    /**
+     * Define a value paginate rows
+     */
+    const ROW_LIMIT = 10;
+
+    /**
+     * Define  value status of reservation
+     */
+    const STATUS_ACCEPT = 1;
+    const STATUS_REJECT = 2;
+    const STATUS_CANCEL = 3;
+
```

Comment:
+    const STATUS_ACCEPT = 1;
 +    const STATUS_REJECT = 2;
 +    const STATUS_CANCEL = 3;

We should change to is better:
+    const STATUS_ACCEPTED = 1;
 +    const STATUS_REJECTED = 2;
 +    const STATUS_CANCELED = 3;